source/wp-content/themes/wporg-news-2021/functions.php

```diff
@@ -213,6 +214,25 @@ function offset_paginated_index_posts( $query ) {
 	$query->set( 'offset', $default_offset + $posts_on_front_page );
 }
 
+/**
+ * Adjust posts_per_page separately for certain categories.
+ *
+ * We have certain category templates designed for different numbers of posts per page. Unfortunately it's not possible to set this in the block temlates.
+ * (See https://github.com/WordPress/wporg-news-2021/issues/70#issuecomment-996460735)
+ *
+ * @param WP_Query $query
+ */
+function category_posts_per_page( $query ) {
```

Comment:
We could give this function a more generic name, in case we need to add more filters in the future. Not a big deal though.

Reply:
Yeah good suggestion, I'll update it. Perhaps something like `override_category_query_args()`.

I did wonder if this should be combined with (or call?) `offset_paginated_index_posts()`, since they're somewhat similar and could interact. Probably simplest to keep them separate unless there's a specific reason to do otherwise.

Reply:
👍🏻